src/solver/conv_mlir_igemm_fwd_xdlops.cpp

```diff
@@ -65,6 +65,8 @@ bool ConvMlirIgemmFwdXdlops::IsApplicable(const ConvolutionContext& ctx) const
         return false;
     if(!IsComposableKernelSupportedHardware(ctx))
         return false;
+    if(ctx.IsInt8())
+        return false;
```

Comment:
Can we remove this? This condition should be tested by `MiirIsConfigApplicable(...)`, one line below.

Reply:
The `MiirIsConfigApplicable` is relying on a string definition of the context, so we would need to pass in the context as an additional argument to be able to call the `IsInt8` method which seems like shoe-horning something into an otherwise MLIR specific method?